docs/wallet/dlc.md

```diff
@@ -63,8 +64,8 @@ val loseHash = CryptoUtil.sha256(ByteVector("LOSE".getBytes)).flip
 
 (pubKey.bytes ++ rValue.bytes).toHex
 (winHash.bytes ++ Satoshis(100000).bytes ++ loseHash.bytes ++ Satoshis.zero.bytes).toHex
-Schnorr.signWithNonce(winHash.bytes, privKey, nonce).hex
-Schnorr.signWithNonce(loseHash.bytes, privKey, nonce).hex
+NativeSecp256k1.schnorrSignWithNonce(winHash.bytes.toArray, privKey.bytes.toArray, nonce.bytes.toArray)
+NativeSecp256k1.schnorrSignWithNonce(loseHash.bytes.toArray, privKey.bytes.toArray, nonce.bytes.toArray)
```

Comment:
nit: `privKey.schnorrSignWithNonce(_Hash.bytes, nonce)`

Reply:
So I don't think this works as these methods take a `ECPRivateKey` and not a `SchnorrNonce` 

```scala
  def schnorrSignWithNonce(
      dataToSign: ByteVector,
      nonce: ECPrivateKey): SchnorrDigitalSignature
```